Add blog post: Introducing YakShaver Anywhere (#760)
SSWConsulting/SSW.Products: content/blogs/YakShaver/yakshave-on-browser.mdx

---
seo:

>   title: Introducing YakShave on Browser!

Reviewer: Rename per the naming decision (see review body):

```suggestion
  title: Introducing YakShaver Anywhere!
```

  description: >-
    Record, narrate, and shave issues into ready-to-work-on work items straight
    from your browser — no desktop app, no extension, nothing to install.
category: What's New

> title: Introducing YakShave on Browser!

Reviewer: ```suggestion
title: Introducing YakShaver Anywhere!
```

date: '2026-07-05T13:00:00.000Z'
author: Steven Qiang
authorImage: /people/Steven-Qiang.jpg
sswPeopleLink: 'https://www.ssw.com.au/people/steven-qiang/'

> bannerImage: /YakShaver/Blogs/yakshave-on-browser.png

Reviewer: Banner path for the renamed post (the banner itself needs re-rendering with the new headline, and uploading to Tina media):

```suggestion
bannerImage: /YakShaver/Blogs/yakshaver-anywhere.png
```

readLength: 2 min
summaryCard: true
summary: >
  ## Shave Issues Straight from Your Browser

>   YakShave on Browser lets you record your screen, narrate the problem, and get

Reviewer: ```suggestion
  YakShaver Anywhere lets you record your screen, narrate the problem, and get
```

  an AI-drafted work item — all without installing anything.

  ***

  ## How It Works

  * Pick your project and hit Record — share a screen, window, or tab

  * Show the problem and talk through it, just like explaining it to a teammate

  * Optionally add written notes, then submit

  * Watch the AI agent analyse your recording and draft the work item live

  ***

  ## Why You'll Love It

  * Zero setup — works on any OS with a modern browser

  * Record your screen, film with your camera, or upload an existing video

  * The agent follows your project's skill, so work items match your team's
  standards
---

> Spotted a bug but don't have YakShaver installed? No problem. With **YakShave on Browser**, you can now shave issues directly from your browser — no desktop app, no extension, nothing to install!

Reviewer: ```suggestion
Spotted a bug but don't have YakShaver installed? No problem. With **YakShaver Anywhere**, you can now shave issues directly from your browser — no desktop app, no extension, nothing to install!
```

> ![](/YakShaver/Blogs/Blogs/yakshave-now-button.png)

Reviewer: The path has `Blogs/` twice so this 404s (it renders as a giant empty box on the preview) — and it needs alt text:

```suggestion
![The New Shave button in the top right of the YakShaver dashboard](/YakShaver/Blogs/yakshave-now-button.png)
```

Note the image also needs uploading to Tina Cloud media or it will still 404 after merge (see review body).

Figure: Find the "New Shave" button

## How it works

> 1. Click on the right top "New Shave" button, select "Shave in browser"

Reviewer: Minor wording fix ("right top" → "top right") and bolding to match the other steps:

```suggestion
1. Click the **New Shave** button in the top right, then select **Shave in browser**
```

2. Pick your project and hit **Record** — choose the screen, window, or tab you want to share, and a short countdown gives you time to get ready
3. Show the problem and talk through it, just like you would explain it to a teammate
4. Optionally add written notes for extra context, then submit

From there, the AI agent takes over. It analyses your recording, follows your project's skill (your team's instructions for how work items should be written), and drafts the work item for you. Best of all, you can **watch the agent work live** — investigating, reasoning, and writing in real time, right in your browser.

## Why you'll love it

* **Zero setup** — works on macOS, Windows, and Linux in any modern browser
* **More ways to capture** — record your screen, film with your camera, or upload an existing video
* **Work items done right** — the agent follows your project's skill, so every item lands in your backlog meeting your team's standards

## Ready to shave?

> Log in at [yakshaver.ai](https://yakshaver.ai), open your project, and hit **Record**. Your next bug report is one recording away. 🐂

Reviewer: The CTA says "hit Record", but the steps above teach New Shave → Shave in browser — suggest aligning them:

```suggestion
Log in at [yakshaver.ai](https://yakshaver.ai), hit **New Shave**, and choose **Shave in browser**. Your next bug report is one recording away. 🐂
```
